app/components/Utility/EquivalentPrice.jsx

```diff
@@ -133,11 +133,11 @@ class EquivalentPrice extends MarketStatsCheck {
         if (marketStats.get(toMarket) && marketStats.get(toMarket).price) {
             toPrice = marketStats.get(toMarket).price.clone();
         }
-
         if (toAsset.get("id") === fromAsset.get("id")) return 1;
 
         let finalPrice;
         if (toPrice && fromPrice) {
+            if(fromPrice.base.amount == 0) return null;
```

Comment:
Might as well add `|| toPrice.base.amount == 0)` as well no?

Reply:
I guess it wouldn't hurt, but toAsset would be the BTS-bitAsset, like butUSD/bitEUR/BITCOIN, which I would hope always have trading.

It would be a future failsafe incase an asset runs out of buy orders, or an asset pair is added where toAsset isn't yet well traded.